lib/PuppeteerSharp/Page.cs

```diff
@@ -101,6 +121,12 @@ private Page(Session client, Target target, FrameTree frameTree, bool ignoreHTTP
 
         #region Public Methods
 
+        public async Task<Dictionary<string, decimal>> MetricsAsync()
```

Comment:
could we use a strongly typed object instead of a `Dictionary<string, decimal>`?

Reply:
This is different. It is a list. This could be a dynamic list of metrics, and I think that for the user it'd be super easy to do something like `metrics["myMetric"]`

Reply:
it's an object, not a list ;)
I just looked at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/puppeteer/index.d.ts#L554

Reply:
Well, that's not v1.0 :p